client/signup/steps/jpo-summary/index.jsx

```diff
@@ -0,0 +1,212 @@
+
+class JPOSummaryStep extends React.Component {
+
+	static propTypes = {
+		flowName: PropTypes.string,
+		goToNextStep: PropTypes.func.isRequired,
+		positionInFlow: PropTypes.number,
+		setJPOSummary: PropTypes.func.isRequired,
+		signupProgress: PropTypes.array,
+		stepName: PropTypes.string
+	};
+
+	constructor( props ) {
+		super( props );
+		this.getFormattedPayload = this.getFormattedPayload.bind( this );
+		this.completeOnboarding = this.completeOnboarding.bind( this );
+		this.renderStepContent = this.renderStepContent.bind( this );
+	}
+
+	getFormattedPayload() {
+		const payload = this.props.signupProgress;
+		return {
+			siteTitle: get( payload[ 0 ], [ 'jpoSiteTitle', 'siteTitle' ], '' ),
+			siteDescription: get( payload[ 0 ], [ 'jpoSiteTitle', 'siteDescription' ], '' ),
+			businessPersonal: get( payload[ 1 ], [ 'jpoSiteType', 'businessPersonal' ], '' ),
+			genre: get( payload[ 1 ], [ 'jpoSiteType', 'genre' ], '' ),
+			businessName: get( payload[ 1 ], [ 'jpoSiteType', 'addressInfo', 'businessName' ], '' ),
+			businessAddress: get( payload[ 1 ], [ 'jpoSiteType', 'addressInfo', 'streetAddress' ], '' ),
+			businessCity: get( payload[ 1 ], [ 'jpoSiteType', 'addressInfo', 'city' ], '' ),
+			businessState: get( payload[ 1 ], [ 'jpoSiteType', 'addressInfo', 'state' ], '' ),
+			businessZipCode: get( payload[ 1 ], [ 'jpoSiteType', 'addressInfo', 'zipCode' ], '' ),
+			homepageFormat: get( payload[ 2 ], 'jpoHomepage', '' ),
+			addContactForm: get( payload[ 3 ], 'jpoContactForm', '' )
+		};
+	}
+
+	completeOnboarding() {
+		// Get the payload and original JPC url
+		const payload = this.getFormattedPayload();
+
+		// Flag the flow as complete for use in JPC
+		localStorage.setItem( 'jpoFlowComplete', '1' );
+
+		// Store the payload in localStorage for use after Jetpack is connected
+		localStorage.setItem( 'jpoPayload', JSON.stringify( payload ) );
+
+		// Redirect to the original JPC URL
+		page.redirect( '/jetpack/connect/plans/' + get( this.props.signupDependencies, [ 'jpoConnect', 'siteSlug' ] ) );
+	}
+
+	renderStepContent() {
+		const connectionToJetpackComplete = true;
+
+		const checkMark = <Gridicon
+			className="jpo-summary__item-checkmark"
+			size={ 12 }
+			icon="checkmark"
+			/>;
+
+		const {
+			signupProgress: {
+				0: {
+					jpoSiteTitle
+				},
+				1: {
+					jpoSiteType
+				},
+				2: {
+					jpoHomepage
+				},
+				3: {
+					jpoContactForm
+				}
+			}
+		} = this.props;
+
+		const pathPrefix = '/start/jetpack-onboarding/';
+
+		return (
+			<div className="jpo__summary-wrapper">
+				<table>
+					<tbody>
+						<tr>
+							<td>
+								<div className="jpo__summary-col-header jpo-summary__col-header">
+									{
+										translate( "Steps you've completed:" )
+									}
+								</div>
+								<ul className="jpo-summary__completed-onboarding">
+									{
+										jpoSiteTitle
+											? <li>{ checkMark } <span>{ translate( 'Site Title & Description' ) }</span></li>
```

Comment:
Why are we wrapping the translation in a span? They are unnecessary inside of an `li`.